models/response/transaction.go

```diff
@@ -0,0 +1,39 @@
+package response
+
+import "github.com/bitcoin-sv/spv-wallet/models/common"
+
+// Transaction is a model that represents a transaction.
+type Transaction struct {
+	// Model is a common model that contains common fields for all models.
+	common.Model
```

Comment:
same as in draft transaction, this should probably be some other Model.

Reply:
replaced with a new model